odbc/src/api/col_attr_describe_tests.rs

```diff
@@ -440,7 +440,7 @@ mod unit {
             (Desc::DisplaySize, 0),
             (Desc::FixedPrecScale, 0),
             (Desc::Length, 0),
-            (Desc::Nullable, 1),
```

Comment:
Here, I was reporting the wrong values before because `ColumnNullability` was not ordered the same as the underlying values, whoops.